1.0.6 test failure on Windows

### Context
I'm attempting to compile and test EvoSuite 1.0.6 prior to making any changes, but currently getting the same test failure as described in #10 (except I'm compiling on Windows). I've checked permissions and the user, as implied by that issue, but I don't see anything amiss. Are there any known problems with testing EvoSuite on Windows?

OS: Win 10 v1803(17134.165)
EvoSuite: 1.0.6
Java
>java version "1.8.0_181"
>Java(TM) SE Runtime Environment (build 1.8.0_181-b13)
>Java HotSpot(TM) 64-Bit Server VM (build 25.181-b13, mixed mode)

### Steps to Reproduce
```
cd C:\path\to\src
mvn clean
mvn test
```

### Current Result
```
[ERROR] Errors:
[ERROR]   MockJFileChooserTest.testGetCurrentDirectory:57 NullPointer
[INFO]
[ERROR] Tests run: 123, Failures: 0, Errors: 1, Skipped: 4
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] EvoSuite 1.0.6 ..................................... SUCCESS [  0.281 s]
[INFO] evosuite-runtime ................................... FAILURE [01:00 min]
[INFO] evosuite-client .................................... SKIPPED
[INFO] evosuite-master .................................... SKIPPED
[INFO] evosuite-plugins ................................... SKIPPED
[INFO] Maven Plugin for EvoSuite .......................... SKIPPED
[INFO] evosuite-maven-plugin-test ......................... SKIPPED
[INFO] build-support-test ................................. SKIPPED
[INFO] evosuite-standalone-runtime ........................ SKIPPED
[INFO] evosuite-shaded .................................... SKIPPED
[INFO] evosuite-generated 1.0.6 ........................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:01 min
[INFO] Finished at: 2018-08-06T15:17:58+01:00
[INFO] ------------------------------------------------------------------------
```

--- a/runtime/src/main/java/org/evosuite/runtime/mock/javax/swing/filechooser/MockFileSystemView.java
+++ b/runtime/src/main/java/org/evosuite/runtime/mock/javax/swing/filechooser/MockFileSystemView.java
@@ -529,10 +529,19 @@ class MockWindowsFileSystemView extends MockFileSystemView {
 	}
 
 	/**
-	 * @return the Desktop folder.
+	 * @return the home directory on the drive the code exists on.
 	 */
 	public File getHomeDirectory() {
-		return getRoots()[0];
+		File executionPath = createFileObject(System.getProperty("user.dir"));
+		File[] roots = getRoots();
+
+		for (File root : roots) {
+			if(root.toPath().getRoot().equals(executionPath.toPath().getRoot())) {
+				return root;
+			}
+		}
+
+		return roots[0];
 	}
 
 	/**

